test/assertInterpolation.js

```diff
@@ -0,0 +1,64 @@
+/* global assert suite test */
+
+function assertInterpolation (transformation, expectation) {
+  var target = document.createElement('div');
+
+  for (var {at, is} of expectation) {
+    var timing = {duration: 1, fill: 'forwards'};
+    var currentTime = at;
+
+    var animation;
+    if (transformation.property === 'translate') {
+      animation = target.animate({translate: [transformation.from, transformation.to]}, timing);
+    } else if (transformation.property === 'rotate') {
+      animation = target.animate({rotate: [transformation.from, transformation.to]}, timing);
+    } else if (transformation.property === 'scale') {
+      animation = target.animate({scale: [transformation.from, transformation.to]}, timing);
+    }
+
+    animation.currentTime = currentTime;
+    var result = target.style._getAnimated(transformation.property);
+
+    animation.cancel();
+    target.remove();
```

Comment:
The target was never attached to the document. I guess that was never a requirement of the web-animations-js polyfill!